src/libs/actions/Policy/Policy.ts

```diff
@@ -1968,6 +1967,7 @@ function buildPolicyData(options: BuildPolicyDataOptions = {}) {
         lastUsedPaymentMethod,
         areDistanceRatesEnabled,
         adminParticipant,
+        selectedFeatures,
```

Comment:
```suggestion
        selectedFeatures: selectedFeatures ? JSON.stringify(selectedFeatures) : undefined,
```
This way you preserve the array of string when you send the payload the api layer. Basically the payload to api layer should be `["one", "two", "three"]` and not `"one, two, three"`